sys-libs/fehqlibs/fehqlibs-10a.ebuild

```diff
@@ -0,0 +1,29 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit multilib
+
+DESCRIPTION="State-of-the-art C routines provided as easy-to-use library for Internet service"
+HOMEPAGE="https://www.fehcom.de/ipnet/qlibs.html"
+SRC_URI="https://www.fehcom.de/ipnet/fehQlibs/fehQlibs-${PV}.tgz"
+
+LICENSE="public-domain"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~m68k ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86 ~x86-fbsd"
```

Comment:
Sorry, but you can only add amd64 and x86, and even those require you to test it with named arches. More exotic arches can only be added by a member of arch team project, so you'll have to request keywording through bugzilla once the ebuild gets merged.

What it basically means is that you'll also have to mask the update of ucpi-ssl on these arches that is missing on fehqlibs, then unmask them when arch teams are done keywording this package.